tests/test.ts

```diff
@@ -0,0 +1,93 @@
+import { assert, expect } from "chai"
+import Club from "../models/club"
+import DbProvider from "../backend_tools/db_provider"
+import { config } from "dotenv"
+import Firebase from "../backend_tools/firebase"
+import User from "../models/user"
+import Year from "../models/year"
+import Role from "../models/role"
+import Career from "../models/career"
+import DbTestProvider from "../backend_tools/test_db_provider"
+
+
+// const firebaseConfig = {
```

Comment:
If the commented code is completely unneeded, it may be good to remove it

Reply:
i removed it